vts/earsigner/README.md

```diff
@@ -0,0 +1,8 @@
+## Configuration
+
+- `ear-signer`: stanza containing the configuration details about the attestation
+  result signing process.  The supported directives are:
+  - `alg`: the [JWS algorithm](https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms)
+    used for signing, e.g.: `ES256`, `RS512`.
+  - `key`: file path of the private key to be used with `alg`.
```

Comment:
Is this a file path or a file (with its full location path) for the private key?

Could you please clarify it a bit?